tests/integration-tests/tests/test_scaling/test_multiple_jobs_submission/cluster-check.sh

```diff
@@ -0,0 +1,265 @@
+#!/bin/bash
```

Comment:
Is this the same the file before, just moved?

Reply:
Exactly the same. Did't do a git move since I'm still preserving the old release-check